src/platform/Linux/bluez/Helper.cpp

```diff
@@ -1718,44 +1721,64 @@ bool BluezUnsubscribeCharacteristic(BLE_CONNECTION_OBJECT apConn)
 
 // ConnectDevice callbacks
 
+using ConnectParams = std::pair<BluezDevice1 *, BluezEndpoint *>;
+
 static void ConnectDeviceDone(GObject * aObject, GAsyncResult * aResult, gpointer)
 {
     BluezDevice1 * device = BLUEZ_DEVICE1(aObject);
     GError * error        = nullptr;
     gboolean success      = bluez_device1_call_connect_finish(device, aResult, &error);
 
-    VerifyOrExit(success == TRUE, ChipLogError(DeviceLayer, "FAIL: ConnectDevice : %s", error->message));
+    if (success != TRUE)
```

Comment:
does `if (!success)` work here or is gboolean not using 0 is false convention?

Reply:
It does. I changed it.